File: app/src/main/java/com/example/multidagger/di/ServiceBuildersModule.java
package com.example.multidagger.di;

import com.example.multidagger.service.RequestService;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
abstract class ServiceBuildersModule {

    @ContributesAndroidInjector
    abstract RequestService contributeRequestService();
}

File: app/src/main/kotlin/com/example/multidagger/di/ServiceBuildersModule.kt
package com.example.multidagger.di

import com.example.multidagger.service.RequestService

import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
internal abstract class ServiceBuildersModule {

    @ContributesAndroidInjector
    internal abstract fun contributeRequestService(): RequestService
}
